Program.fs

```diff
@@ -148,25 +137,34 @@ module MainModule =
                         tally = tally
                     }
             }
-        let! response = buildMessage Http.HttpMethod.Post url (Some header) (Some message) |> makeRequest
+        let! response =
+            buildMessage Http.HttpMethod.Post url (Some header) (Some message)
```

Comment:
For longer lines, it's better to wrap the pipe chains to make the "flow" of values more visible. I have done this here in a few (not all) places to show the effect.